src/cli/export/groups_test.go

```diff
@@ -71,6 +71,7 @@ func (suite *GroupsUnitSuite) TestAddGroupsCommands() {
 
 				// bool flags
 				"--" + flags.ArchiveFN,
+				"--" + flags.RawFN,
```

Comment:
I was thinking we could add something like `--format raw`. This could be later expanded to `--format html` or something for an html export of the chats. These would be similar for exchange where the formats would be `pst`, `mbox` etc.

Related: https://github.com/alcionai/corso/blob/edf753382ea99cb255f0ac1747adb97dcd5b8069/src/pkg/control/export.go#L15

Reply:
sure, sounds good to me